beta-20160629.md

```diff
@@ -0,0 +1,60 @@
+---
+title: What's New in beta-20160629
+toc: false
+summary: Additions and changes in CockroachDB version beta-20160630.
+---
+
+## Jun 29, 2016
+
+### New Features
+
+- A prototype implementation of `JOIN` (non-optimized) is now available. [#7202](https://github.com/cockroachdb/cockroach/pull/7202)
+- [Column Families](https://github.com/cockroachdb/cockroach/blob/master/docs/RFCS/sql_column_families.md) are a new, more efficient key-value representation of SQL tables. The [FAMILY](column-families.html) syntax extension allows users to tune column families, when necessary. [#7466](https://github.com/cockroachdb/cockroach/pull/7466), [#7408](https://github.com/cockroachdb/cockroach/pull/7408)
```

Comment:
No data migration is necessary: the previous format corresponds to a separate column family for each column in the table.